mermithid/processors/misc/KrComplexLineShape.py

```diff
@@ -434,7 +453,7 @@ def fit_data(self, freq_bins, data_hist_freq, print_params=True):
         }
         return dictionary_of_fit_results
 
-    def make_spectrum_1(self, gauss_FWHM_eV, prob_parameter, emitted_peak='shake'):
+    def make_spectrum_1(self, prob_parameter, emitted_peak='shake'):
```

Comment:
give functions more descriptive names than make_spectrum_N. Also please add comment with description for each function.